Tests/DependencyInjection/MonologExtensionTest.php

```diff
@@ -562,6 +602,50 @@ public function testFingersCrossedHandlerWhenExcludedHttpCodesAreSpecified()
         $this->assertDICConstructorArguments($handler, [new Reference('monolog.handler.nested'), new Reference('monolog.handler.main.http_code_strategy'), 0, true, true, null]);
     }
 
+    public function testFingersCrossedHandlerWhenExcludedHttpCodesAreSpecified()
+    {
+        if (!class_exists(HttpCodeActivationStrategy::class)) {
+            $this->markTestSkipped('Symfony Monolog 4.1+ is needed.');
+        }
+        if (!\class_exists(SwitchUserTokenProcessor::class)) {
+            $this->markTestSkipped('Symfony Monolog 5.2+ is needed.');
+        }
```

Comment:
Not sure why fabbot.io is not running, but either we use class_exists with or without a leading backslash 🧐 can you please check or reopen the PR to trigger fabbot again?

I am currently on a phone 📱 

Thanks

Reply:
Reopening does not trigger fabbot, apparently.

Reply:
generally, it will work in both cases, but `\class_exists()` will be slightly faster, as parser will not check current namespace for such function and will check global namespace right away.

but in terms of code style consistency, IMO there shouldn't be such mix of usage. Either with backslash, either without ;)